django/core/management/commands/inspectdb.py

```diff
@@ -48,7 +48,7 @@ def table2model(table_name):
             yield "# You'll have to do the following manually to clean this up:"
             yield "#   * Rearrange models' order"
             yield "#   * Make sure each model has one field with primary_key=True"
-            yield "#   * Make sure each ForeignKey has `on_delete` set to the desired behavior."
+            yield "#   * Make sure each ForeignKey and OneToOneField has `on_delete` set to the desired behavior"
```

Comment:
Trailing dot is missing.

Reply:
This is on purpose. No others in the same bulleted list use it either, and this isn't the last item in it.